helpers/EmailVerification.js

```diff
@@ -0,0 +1,36 @@
+import Postals from './Postals';
+
+/**
+ *
+ *
+ * @export
+ * @class EmailVerification
+ */
+export default class EmailVerification {
+  /**
+  *
+  * @static
+  * @param {string} requestInfo - request information
+  * @param {string} recipient - recipient of the email
+  * @param {string} name - name of the recipient
+  * @param {string} token - verification token
+  * @memberof EmailVerification
+  * @returns {function} - returns a function call
+  */
+  static async sendVerificationEmail({ protocol, hostname }, recipient, name, token) {
```

Comment:
Beautiful destructuring here! I love it!

Reply:
Thank you very much